assets/css/common/post-single.css

```
@@ -701,6 +701,56 @@ codapi-snippet codapi-toolbar button:after {
+    user-select: none;
+}
+
+@media (hover: hover) {
+    .tabs__tab:hover {
+        color: var(--primary);
+    }
+}
+
+.tabs__tab.is-active {
+    color: var(--primary);
+    border-bottom-color: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)